src/modules/route_request.js

```diff
@@ -19,4 +19,4 @@ const getRoute = {
   },
 };
 
-export default getRoute;
+export default calculateRoute;
```

Comment:
change this to `export { name }` and then also change the import where you have it to `import { name } from...` that is the standard with modules